dams-tests/tests/common.rs

```diff
@@ -142,8 +140,19 @@ async fn server_test_config() -> dams::config::server::Config {
     .fold("\n[[service]]".to_string(), |acc, (key, value)| {
         format!("{}\n{} = \"{}\"", acc, key, value)
     });
+    // Format string for database information
+    let database = HashMap::from([
+        ("mongodb_uri", "mongodb://localhost:27017"),
+        ("db_name", "dams-test-db"),
+    ])
+    .into_iter()
+    .fold("\n[database]".to_string(), |acc, (key, value)| {
+        format!("{}\n{} = \"{}\"", acc, key, value)
+    });
```

Comment:
I'm gonna be honest, I really like this kind of functional wizardry. It always makes me happy.

I think something like this might be more clear though:

```suggestion
    let database = r#"
        [database]
        mongodb_uri = "mongodb://localhost:27017"
        db_name = "dams-test-db"
    "#;
```